testcontainers/compose.py

```diff
@@ -204,15 +204,30 @@ def get_service_host(self, service_name, port):
         str:
             The hostname for the service
         """
-        return self._get_service_info(service_name, port)[0]
+        return self.get_service_info(service_name, port)[0]
 
-    def _get_service_info(self, service, port):
+    def get_service_info(self, service, port):
+        """
+        Returns the host and the port for one of the services.
+
+        Parameters
+        ----------
+        service: str
+            Name of the docker compose service
+        port: int
+            The internal port to get the host for
+
+        Returns
+        -------
+        (str, str):
+            The hostname for the service, The port for the service
+        """
         port_cmd = self.docker_compose_command() + ["port", service, str(port)]
         output = subprocess.check_output(port_cmd, cwd=self.filepath).decode("utf-8")
         result = str(output).rstrip().split(":")
         if len(result) != 2 or not all(result):
             raise NoSuchPortExposed(f"port {port} is not exposed for service {service}")
-        return result
+        return result[0], result[1]
```

Comment:
```suggestion
        return result
```

Shall we keep this as is, avoiding the extra indexing?

Reply:
I only did this so the return type would be a `(str, str)` string touple and not a string array

Reply:
I was playing with the idea to introduce a `HostPort = collections.namedtouple("HostPort", ["host", "port])` and wrap the result there

Reply:
Let's keep this as is for now unless there's a particular reason to return a tuple? If a tuple is required, let's use `tuple(result)`.

Reply:
@tillahoffmann I commited your suggestions. Can you please approve :)